voter-dapp/README.md

```diff
@@ -1,6 +1,71 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Voter dApp
 
-## Available Scripts
+This is a browser client that UMA token holders can use to vote on price requests. The dApp interacts with the deployed [DVM](https://docs.umaproject.org/uma/oracle/technical_architecture.html) code using MetaMask's browser extension. The source code was bootstrapped from [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Assuming there were no changes to the voter-dapp specific README, and that changes to it are outside scope of this PR?

Reply:
I did make some very minor updates to it to reflect its new location and updated commands in prerequisites. Do you think I should copy it over exactly and save those updates for a separate PR?

Reply:
Oh no not at all, can you highlight the changes possibly?

Reply:
Will do!